ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/index.scss

```diff
@@ -3,4 +3,8 @@
     font-size: $font-size-h4;
     margin: 0 8px 0 8px;
   }
+
+  label {
```

Comment:
What do you think about using `&__label`? Thinking that if AdvancedGasFeeDefaults ever uses a component that itself uses a `label` (without you knowing) it might be good to prevent this style from getting accidentally applied to that label.

Reply:
Good point, I just updated the PR with fix.